### PR TITLE
Spike/sc 1929

### PR DESCRIPTION
**Pull request by abnepc**
## Jira Story
[SC-1929](https://epcpower.atlassian.net/browse/SC-1929)
## About
Investigate upgrading from `python3.7`
## Associated Pull Requests
* [sunspec-demo](https://github.com/epcpower/sunspec-demo/pull/34)
* [stlib](https://github.com/epcpower/stlib/pull/47)
* [pm](https://github.com/epcpower/pm/pull/92)
* [grid-tied](https://github.com/epcpower/grid-tied/pull/664)
* [rm-dc](https://github.com/epcpower/rm-dc/pull/20)
* [m-bcu](https://github.com/epcpower/m-bcu/pull/6)
* [m-pm](https://github.com/epcpower/m-pm/pull/3)
* [m-tcu](https://github.com/epcpower/m-tcu/pull/44)

## Reproduction Steps
N/A
## Validation
Check that CI is using `python3.8`

[SC-1929]: https://epcpower.atlassian.net/browse/SC-1929?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -61,6 +61,7 @@ PyQt5 = "5.14.1"
 requests = "2.27.1"
 canmatrix = "0.9.1"
 click = "7.1.2"
+pylint-runner = "^0.6.0"
```
Comment (gordon-epc) [inline pyproject.toml:64]: This shouldn't be here.
```suggestion
```

File: pyproject.toml [hunk 2]
```diff
@@ -47,12 +47,12 @@ attrs = ">=19.3.0"
 natsort = "5.5.0"
 arrow = "0.12.1"
 PyQtChart = "5.14.0"
-pywin32 = { version = "223", markers = "sys_platform == 'win32'" }
+pywin32 = { version = "225", markers = "sys_platform == 'win32'" }
```
Comment (gordon-epc) [inline pyproject.toml:50]: I assume pywin32 upgrade from 223 to 225 is just to get it to work with Python 3.8?

**Review by gordon-epc — CHANGES_REQUESTED** [review]
Check readme for additional changes.

File: pyproject.toml (hunk 2 above)
Comment (abnepc) [reply pyproject.toml:50]: Yes, `pywin32` first supports `python3.8` in version `225`

File: pyproject.toml (hunk 1 above)
Comment (abnepc) [reply pyproject.toml]: [4167a7f](https://github.com/epcpower/st/pull/48/commits/4167a7f05064dd583fa52044ee03ec872996049e)

**Review by gordon-epc — APPROVED** [review]

**Pull request merged**